browser/ui/BUILD.gn

```diff
@@ -153,6 +154,7 @@ source_set("ui") {
     "//brave/browser/tor",
     "//brave/common",
     "//brave/components/brave_adblock_ui:generated_resources",
+    "//brave/components/brave_gcm_driver:brave_gcm_driver",
```

Comment:
`:brave_gcm_driver` is redundant. `//brave/components/brave_gcm_driver` is all you need